flow/flow.go

```diff
@@ -104,20 +104,20 @@ const (
 )
 
 type Custom struct {
-	ID   uint32
-	Type Type
 	Str  string
-	U8   byte
-	U16  uint16
-	U32  uint32
 	U64  uint64
-	I8   int8
-	I16  int16
-	I32  int32
+	Type Type
```

Comment:
Let's add a top level comment on this struct to explicitly mentioning the ordering is done with struct padding in mind having 64-bit boundaries. This will help make it known that we've done this intentionally in an explicit way to avoid possible regressions of moving things around and so that we know when adding new fields that we should be conscious of the field ordering.